PocketKit/Sources/PocketKit/Main/MainCoordinator.swift

```diff
@@ -29,7 +29,7 @@ class MainCoordinator {
         regular.setCompactViewController(compact.viewController)
 
         model.$selectedSection
-            .sink { section in
+            .receive(on: DispatchQueue.global(priority: .background)).sink { section in
```

Comment:
This performs an analytic action and was receiving it on the main thread. This should be in a background thread.

Reply:
@bassrock I understand making this explicit, but does the underlying tracker (Snowplow) automatically do this on a background thread? I feel like the answer is "maybe". Did something show up in Xcode to suggest this change?

Reply:
@dskuza Good call just looked, it does. I reverted this.